grafana-plugin/src/containers/IntegrationForm/IntegrationForm.styles.ts

```diff
@@ -0,0 +1,64 @@
+import { css } from '@emotion/css';
+import { GrafanaTheme2 } from '@grafana/data';
+
+export const getIntegrationFormStyles = (theme: GrafanaTheme2) => {
+  return {
+    form: css`
```

Comment:
Can we stick to a single way of using `css` (object syntax) as suggested [here](https://github.com/grafana/grafana/blob/main/contribute/style-guides/styling.md#usage)?